daffodil-cli/src/it/scala/org/apache/daffodil/parsing/TestCLIParsing.scala

```diff
@@ -1300,28 +1300,6 @@ class TestCLIparsing {
     }
   }
 
-  @Test def test_XXX_CLI_Parsing_SimpleParse_exi(): Unit = {
-
-    val schemaFile = Util.daffodilPath("daffodil-test/src/test/resources/org/apache/daffodil/section00/general/generalSchema.dfdl.xsd")
-    val (testSchemaFile) = if (Util.isWindows) (Util.cmdConvert(schemaFile)) else (schemaFile)
-
-    val shell = Util.start("")
-
-    try {
-      val cmd = String.format(Util.echoN("Hello") + "| %s parse -I exi -s %s -r e1 | md5sum", Util.binPath, testSchemaFile)
-
-      shell.sendLine(cmd)
-      shell.expect(contains("937b3f96ee0b5cd1ac9f537cf8ddc580"))
-
-      Util.expectExitCode(ExitCode.Success, shell)
-
-      shell.send("exit\n")
-      shell.expect(eof)
-    } finally {
-      shell.close()
-    }
-  }
-
```

Comment:
Why remove his EXI test? Isn't this the only test that verifies EXI?

Reply:
There appear to be 4 CLI tests which call -I exi but I'd like to know why the test was removed too.

```
daffodil-cli/src/it/scala/org/apache/daffodil/parsing/TestCLIParsing.scala
1311:      val cmd = String.format(Util.echoN("Hello") + "| %s parse -I exi -s %s -r e1 | md5sum", Util.binPath, testSchemaFile)

daffodil-cli/src/it/scala/org/apache/daffodil/performance/TestCLIPerformance.scala
79:      val cmd = String.format("%s performance -I exi -N 2 -t 2 -s %s -r matrix %s", Util.binPath, testSchemaFile, testInputFile)
207:      val cmd = String.format("%s performance --unparse -I exi -N 2 -t 2 -s %s -r e3 %s", Util.binPath, testSchemaFile, testInputFile)

daffodil-cli/src/it/scala/org/apache/daffodil/unparsing/TestCLIUnparsing.scala
618:      val cmd = String.format("%s unparse -I exi -s %s --root e1 %s", Util.binPath, testSchemaFile, testInputFile)
```